SUFileManager + SULog

1.x (at 6471af1bbff890c7c91e225654af4175d9f99086) doesn't build:

```
Undefined symbols for architecture arm64:
  "_SULog", referenced from:
      -[SUFileManager _removeQuarantineWithAuthenticationAtRootURL:error:] in SUFileManager.o
      -[SUFileManager copyItemAtURL:toURL:error:] in SUFileManager.o
      -[SUFileManager moveItemAtURL:toURL:error:] in SUFileManager.o
      -[SUFileManager changeOwnerAndGroupOfItemAtRootURL:toMatchURL:error:] in SUFileManager.o
      -[SUFileManager updateModificationAndAccessTimeOfItemAtURL:error:] in SUFileManager.o
      -[SUFileManager makeDirectoryAtURL:error:] in SUFileManager.o
      -[SUFileManager removeItemAtURL:error:] in SUFileManager.o
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -39,6 +39,11 @@ jobs:
           DEVELOPER_DIR: ${{ matrix.xcode }}
         run: |
           xcodebuild test-without-building -project Sparkle.xcodeproj -scheme UITests -configuration Debug -derivedDataPath build
+      - name: Build Release Distribution
+        env:
+          DEVELOPER_DIR: ${{ matrix.xcode }}
+        run: |
+          xcodebuild build -project Sparkle.xcodeproj -scheme Distribution -configuration Release -derivedDataPath build
       - name: Archive Test Results
         if: failure()
         uses: actions/upload-artifact@v2

--- a/Sparkle.xcodeproj/project.pbxproj
+++ b/Sparkle.xcodeproj/project.pbxproj
@@ -281,6 +281,7 @@
 		72AC6B2A1B9AAF3A00F62325 /* SparkleTestCodeSignApp.tar.bz2 in Resources */ = {isa = PBXBuildFile; fileRef = 72AC6B291B9AAF3A00F62325 /* SparkleTestCodeSignApp.tar.bz2 */; };
 		72AC6B2C1B9AB0EE00F62325 /* SparkleTestCodeSignApp.tar.xz in Resources */ = {isa = PBXBuildFile; fileRef = 72AC6B2B1B9AB0EE00F62325 /* SparkleTestCodeSignApp.tar.xz */; };
 		72AC6B2E1B9B218C00F62325 /* SparkleTestCodeSignApp.dmg in Resources */ = {isa = PBXBuildFile; fileRef = 72AC6B2D1B9B218C00F62325 /* SparkleTestCodeSignApp.dmg */; };
+		72B3DC8225E3232400131B67 /* SULog.m in Sources */ = {isa = PBXBuildFile; fileRef = 55C14F05136EF6DB00649790 /* SULog.m */; };
 		72C4AB7A25E0B4C500D14562 /* SULog.m in Sources */ = {isa = PBXBuildFile; fileRef = 55C14F05136EF6DB00649790 /* SULog.m */; };
 		72C4ABAB25E0BF1900D14562 /* SULog.m in Sources */ = {isa = PBXBuildFile; fileRef = 55C14F05136EF6DB00649790 /* SULog.m */; };
 		72D4DAA11AD7632900B211E2 /* SUBinaryDeltaCreate.m in Sources */ = {isa = PBXBuildFile; fileRef = 7268AC621AD634C200C3E0C1 /* SUBinaryDeltaCreate.m */; };
@@ -2456,6 +2457,7 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				72B3DC8225E3232400131B67 /* SULog.m in Sources */,
 				61B5F93009C4CFDC00B25A18 /* main.m in Sources */,
 				726F2CEB1BC9C733001971A4 /* SUConstants.m in Sources */,
 				5A6D31EE1BF53245009C5157 /* SUFileManager.m in Sources */,

